server/watson_client.js

```diff
@@ -0,0 +1,8 @@
+var fs = require('fs');
+
+module.exports = function createWatsonClient() {
+  const watsonConfig = (process.env.NODE_ENV === 'development')
+    ? JSON.parse(fs.readFileSync('./tmp/watson_config.json'))
+    : JSON.parse(process.env.WATSON_CONFIG_JSON);
```

Comment:
This probably needs a line in the README.

Reply:
yep! I added this to issue https://github.com/mit-teaching-systems-lab/threeflows/issues/350